src/app/create-event/page.tsx

```diff
@@ -60,6 +66,22 @@ export default function CreateEvent() {
       })
   }
 
+  /* 
+    Add userName to responders array when they click the "Save" button 
+   */
+  const handleSaveResponse = () => {
+    if (userName) {
+      setResponders((prevResponders) => {
+        const updatedResponders = [...prevResponders, userName]
```

Comment:
Future callout that this will need to change to append local to the remote api call that selects all responder usernames to an event instead of relying on the local representation